packages/vitest/package.json

```diff
@@ -146,6 +146,7 @@
     "execa": "^8.0.1",
     "local-pkg": "^0.5.0",
     "magic-string": "^0.30.5",
+    "magicast": "^0.3.2",
```

Comment:
I don't like having magicast in Vitest dependencies. I think it should be defined only in coverage for better size management

Reply:
Ok, we can do that. There's already similar approach for `istanbul-lib-coverage`. `vitest` package has it's types as `devDependencies` and actual package is in coverage providers' `dependencies`. It's then passed from providers to `vitest` side as function arguments when needed.